docs/getting-started.md

```diff
@@ -42,7 +42,7 @@ Users of such distributions may have to install containerd from the source or a
 
 ##### systemd
 If you intend to start containerd via systemd, you should also download the `containerd.service` unit file from
-https://raw.githubusercontent.com/containerd/containerd/main/containerd.service into `/usr/local/lib/systemd/system/containerd.service`,
+https://raw.githubusercontent.com/containerd/containerd/main/containerd.service into `/usr/local/lib/systemd/system/containerd.service` (for RHEL OS distrubaton, Please use this path: `/etc/systemd/system/containerd.service`),
```

Comment:
Why?

Reply:
I confirmed `/usr/local/lib/systemd/system/containerd.service` works for CentOS 7, CentOS Stream 8, and CentOS Stream 9.

Reply:
in Redhat Enterprice 7 this path does not exists.

Reply:
You can just mkdir it, no?